src/Schema/Events/Click.ts

```diff
@@ -2190,4 +2190,34 @@ export interface ClickedCompleteYourProfile {
   context_page_owner_id: string
   context_page_owner_slug?: string
   user_id: string
+}
+
+/**
+ * A user clicks on the header taking them to the app store 
+ *
+ * This schema describes events sent to Segment from [[clickedDownloadAppHeader]]
+ *
+ */
+
+export interface ClickedDownloadAppHeader {
+  action: ActionType.clickedDownloadAppHeader
+  context_page_owner_type: PageOwnerType
+  context_page_owner_id: string
+  context_page_owner_slug?: string
+  user_id: string
```

Comment:
@leodmz 

I think we should change this (also ClickedDownloadAppFooter) to:  

```
  context_page_owner_id?: string
  context_page_owner_slug?: string
  user_id?: string 
 ```
This should be optional because the app download banner is shown also to non-logged-in users, right? 

cc @dzucconi